rosdep/python.yaml

```
@@ -8311,6 +8311,7 @@ python3-pyxdg-pip:
   ubuntu:
     pip:
       packages: [pyxdg]
+python3-pyzbar-pip: *migrate_eol_2025_04_30_python3_pyzbar_pip
```

In my opinion, I think we should do this differently.  In particular, it looks like `python3-pyzbar` is now available in Ubuntu and Debian, since `jammy` and `bullseye`, respectively: https://pkgs.org/search/?q=python3-pyzbar.  So I think we should instead make this a full key:

```suggestion
python3-pyzbar:
  debian: [python3-pyzbar]
  ubuntu:
    '*': [python3-pyzbar]
    focal: null
```

---

I think we should add both.

---

@MatthijsBurgh our current policy is not to add new pip keys for packages that are available in distribution repositories. pip-installed dependencies cannot be used in officially distributed packages. So pip packages in the official sources are only used to backfill dependencies on platforms where they are unavailable in the distribution.

If you specifically need a pip package for some reason the recommendation is now to use custom rosdep sources locally.